juice/examples/benchmarks.rs

```diff
@@ -47,15 +47,15 @@ fn native_backend() -> Rc<Backend<Native>> {
 #[cfg(feature = "cuda")]
 #[allow(dead_code)]
 fn cuda_backend() -> Rc<Backend<Cuda>> {
-    let mut framework = Cuda::new();
+    let framework = Cuda::new();
     let hardwares = &framework.hardwares()[0..1].to_vec();
     println!("Device: {:?}/{}",
              hardwares[0].hardware_type().unwrap(),
              hardwares[0].name().unwrap());
     let backend_config = BackendConfig::new(framework, hardwares);
     let mut backend = Backend::new(backend_config).unwrap();
-    backend.framework.initialise_cublas();
-    backend.framework.initialise_cudnn();
+    backend.framework.initialise_cublas().unwrap();
```

Comment:
it would be awesome if you could add a few comments on why cublas and cudnn _must_ be initialized here and ref #94 #91